SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetConnection.cpp

```diff
@@ -79,8 +79,8 @@ void USpatialNetConnection::LowLevelSend(void* Data, int32 CountBits, FOutPacket
 
 bool USpatialNetConnection::ClientHasInitializedLevelFor(const AActor* TestActor) const
 {
-	check(Driver->IsServer());
-	return true;
+	return ensureAlwaysMsgf(Driver->IsServer(), TEXT("ClientHasInitializedLevelFor should only be called on servers. Actor %s"),
```

Comment:
is it correct to return false here?
aren't the two operations separate?

Reply:
Hard to immediately interpret but will take conservative route to changing back to always return true 👍